source/documentation/adrs/adr-021.html.md.erb

```diff
@@ -0,0 +1,34 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: ADR-021 Management of DNS Records through OctoDNS
+last_reviewed_on: 2024-06-26
+review_in: 6 months
+---
+
+# ADR-021 Managing DNS Records Through OctoDNS
+
+## Status
+
+✅ Accepted
+
+## Context
+
+The Operations Engineering (OE) team at the Ministry of Justice (MoJ) needs to improve DNS management to address security and operational inefficiencies. Current unstructured processes lead to inconsistencies and delays.
+By gathering security stakeholder requirements, the OE team aims to make DNS management more robust, secure, and compliant, reducing manual interventions and aligning practices with MoJ's security policies
+After evaluating OctoDNS and splitting the Terraform state file by hosted zone, we chose OctoDNS for managing Route53 records. This eliminates the state file dependency, simplifies DNS record management with YAML, and integrates with our CI/CD pipelines, enhancing efficiency and maintainability
+
+## Decision
+
+- After conducting Proof of Concepts (PoCs) for both OctoDNS and Terraform, we have decided to use OctoDNS for managing our Route53 DNS records. OctoDNS eliminates the need for a state file, significantly improving efficiency and reducing complexity.
+- Its standardised YAML configuration format is more readable and maintainable, and it integrates seamlessly with our CI/CD pipelines for automated deployment and updates. Performance tests showed that OctoDNS operations are faster, providing a reliable and maintainable solution for our DNS management needs. 
```

Comment:
The point on performance should be first as I think that is the primary benefit; performance is much faster.

Reply:
ok i will change it to first point